index.js

```diff
@@ -140,6 +140,7 @@ export function listenToPopState(historyBus) {
             title:    stateData.title,
             location: stateData.location
         });
+        window.document.title = stateData.title;
```

Comment:
should we be a little defensive here to protect against `null` or `undefined` .... `window.document.title = stateData.title || ''` perhaps? or `window.document.title = stateData.title || window.document.title` may be more appropriate I suppose

Reply:
yeah actually, I like the idea of defaulting back to the current one. I can't really think of any cases where we'd want an empty title